### ISSUE TITLE
operator: default image

### ISSUE THREAD

**Issue by pohly** [intel/pmem-csi#578]
The API description says:
```
image | string | PMEM-CSI docker image name used for the deployment | the same version as the operator |
```

The description of the default is incomplete: it describes the default version, but not the complete image name.

My expectation was that the operator picks up the exact image that was used to deploy it and then uses that.

**Comment by avalluri** [comment]
Finding the exact image is a bit tricky, It has to be provided with commandline argument to the operator in the deployment. Otherwise, we couldn't figureout this from within the operator.

@pohly  do you have any other ideas here?

**Comment by pohly** [comment]
> It has to be provided with commandline argument to the operator in the deployment. 

That was also my first thought.

Alternatively, can the operator figure out in which pod it runs? Then it can retrieve the image from there.

**Comment by avalluri** [comment]
> Alternatively, can the operator figure out in which pod it runs? Then it can retrieve the image from there.

I implemented this solution in #584

**Comment by pohly** [comment]
Fixed.
